Sources/App/Auth/Controllers/AuthController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AuthController.swift
+//  
+//
+//  Created by Joe Williams on 14/11/2021.
+//
+
+import Foundation
+import Vapor
+import Fluent
+
+struct AuthController: RouteCollection {
+    func boot(routes: RoutesBuilder) throws {
+        routes.post("create", use: create)
+        
+        let passwordProtected = routes.grouped(AppUser.authenticator())
+        passwordProtected.post("login", use: login)
+        
+        let adminProtected = routes.grouped(AdminAuthenticatable())
```

Comment:
object responsibilities need firming up here. It probably shouldn't be doing the "admin" stuff, but instead responsible for creating / authenticating / resetting a user.